canonical-web-and-design/ubuntu.com
# Add orange box competition takeover and engage page

## Done

- Takeover and engage page

[brief](https://docs.google.com/spreadsheets/d/19aWd11o6y8ZDHS6NdK4wHDRCsSRAeiTsDdkLSM5ZAAA) | [design](2453) | [Engage copy](https://docs.google.com/document/d/19-Fpi-4XO_pGwukz_ZjSFhmraV58Os42AN4UJJq8FgI/edit)

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/ and refresh until you see the "Quick, easy, on-prem Kubernetes" - test all viewports
- Click the CTA to see the engage page - test all viewports
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)


## Issue / Card

Fixes #7089
